driver/bpf/plumbing_helpers.h

```diff
@@ -16,10 +16,16 @@ or GPL2.txt for full copies of the license.
 #include "types.h"
 #include "builtins.h"
 
-#define _READ(P) ({ typeof(P) _val;				\
-		    memset(&_val, 0, sizeof(_val));		\
-		    bpf_probe_read(&_val, sizeof(_val), &P);	\
-		    _val;					\
+#define _READ(P) ({ typeof(P) _val;					\
+		    memset(&_val, 0, sizeof(_val));			\
```

Comment:
Do we really need to `memset` here?

Reply:
I think we need to clean up all these helpers, we don't need the `memset` since `bpf_probe_read...` automatically do that for us if it is not able to read from the address

Reply:
I will add a commit for those. Thanks a lot.

Reply:
Done w/ https://github.com/falcosecurity/libs/pull/809/commits/1c75bb24e8961e254c9410a227163ed63d7183fc